tests/questionTypes.js

```diff
@@ -451,7 +451,7 @@ define([
 
             it("should remove appearance attribute from label", function () {
                 var from = "Trigger",
-                    to = "Text",
+                    to = "MSelect",
```

Comment:
Trigger -> MSelect wouldn't have passed before this change (that was the bug) and this tests what I thought it would test when I originally wrote it. I can change it to be a more generalized form of test to test more scenarios though